hack/update-deps.sh

```diff
@@ -29,13 +29,13 @@ FLOATING_DEPS=(
 )
 
 REPLACE_DEPS=(
-  "knative.dev/eventing-kafka-broker=github.com/openshift-knative/eventing-kafka-broker@${EVENTING_KAFKA_BROKER_VERSION}"
```

Comment:
reordered to make sure the script runs. Eventing deps need new version of upstream deps first